### PR TITLE
[examples] fix redwood template and update examples to use at least node@16

### PR DESCRIPTION
**Pull request by EndangeredMassa**
The redwood template was broken because it would use node@18, which is not supported for the version of redwood used in the template. This PR updates that version to be node@16, which does work.

While we're at it, I also updated other examples to be at least node@16. I tested deployments of each of these and the all work.

### CODE REVIEW

File: examples/redwoodjs/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "extends": "@redwoodjs/eslint-config"
   },
   "engines": {
-    "node": ">=12",
+    "node": ">=16",
```
Comment (TooTallNate) [inline examples/redwoodjs/package.json]: This would still use Node 18, no?

Comment (EndangeredMassa) [reply examples/redwoodjs/package.json]: ack, forgot to push up that change; thanks!

https://github.com/vercel/vercel/pull/10395/commits/6ba9b2d7a3d429a8199222233d6eb1e3e5f39c07

**Review by TooTallNate — DISMISSED** [review]

**Review by leerob — DISMISSED** [review]

File: examples/umijs/package.json [hunk 2]
```diff
@@ -46,6 +46,6 @@
     ]
   },
   "engines": {
-    "node": ">=8.0.0 <17"
+    "node": ">=16.0.0 <17"
```
Comment (cb1kenobi) [inline examples/umijs/package.json]: **suggestion (non-blocking):** shorthand range?‏

```suggestion
    "node": "16.x"
```

**Review by cb1kenobi — DISMISSED** [review]

**Review by trek — APPROVED** [review]

**Review by cb1kenobi — APPROVED** [review]
LGTM

**Review by leerob — APPROVED** [review]

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**